libs/coin-modules/coin-solana/src/utils.ts

```diff
@@ -23,9 +23,9 @@ export const LEDGER_VALIDATOR_BY_CHORUS_ONE: ValidatorsAppValidator = {
   avatarUrl:
     "https://s3.amazonaws.com/keybase_processed_uploads/3c47b62f3d28ecfd821536f69be82905_360_360.jpg",
   wwwUrl: "https://www.ledger.com/staking",
-  activeStake: 0,
+  activeStake: 10001001000098,
```

Comment:
Are we sure we want to hardcode this value?

Reply:
Yes!